dist/bootstrap4-clockpicker.js

```diff
@@ -0,0 +1,881 @@
+/*!
+ * ClockPicker v0.2.1 (http://weareoutman.github.io/clockpicker/)
+ * Copyright 2014 Wang Shenwei.
+ * Licensed under MIT (https://github.com/weareoutman/clockpicker/blob/gh-pages/LICENSE)
+ * Bootstrap 4 support by djibe
```

Comment:
Hi. I'm really a noob with GitHub so I don't understand anything about your requests. Sorry.
But don't hesitate to guide me to enhance it.
True I didn't test on mobile.
So you can test it here : https://www.diabeclic.com/daemonite/bootstrap-datetimepicker-bs4daemonite.html

Reply:
Hi, no problem. My suggestions were mainly two things:
  * Unify spaces/tabs in the code (i.e. don't use tabs in one line, then spaces in the other; follow what the original code was using)
  * Instead of naming the file `bootstrap4-clockpicker.js`, just overwrite the previous one  (`bootstrap-clockpicker.js`). As yours could be a new version, then users who want to use the bootstrap 3 version can use something <= 0.0.7

Don't waste much time on this, as the owner of the repo is gone and PRs probably won't be merged anyway :(

Thanks for the demo link!